### PR TITLE
Refactor build files

### PR DESCRIPTION
**Pull request by chao-xian**
#### What problem does the pull request solve?

Bringing the codebase closer to Rails convention with how it runs tasks in setting up and running tests.

Trello card:

None

#### Checklist

- [x] I've used the pull request template
- [ ] I've linked this PR to the relevant issue (if mission work)
- [ ] I've written unit tests for these changes (if code change)
- [x] I've updated the documentation in (If any documentation requires updating)
  - [x] README.md
  - [ ] Elsewhere (please link)

### CODE REVIEW

**Comment by chao-xian** [comment]
I realise this makes it out of step with the other repos but I’ll raise PRs for those too.

File: .rspec
Comment (aliuk2012) [inline .rspec:1]: why is this being deleted?

Comment (chao-xian) [reply .rspec:1]: The document format isn't that useful as a default as I doubt most people read it all when tests are passing. Any failing specs will output the documentation anyway. It's just noisy.

Comment (chao-xian) [reply .rspec:1]: (We don't tend to output the document format on govuk for example)

Comment (aliuk2012) [reply .rspec:1]: Makes sense, thanks for the explanation.  👍

**Review by aliuk2012 — APPROVED** [review]

**Comment by chao-xian** [comment]
@aliuk2012 thanks Al! I'll raise corresponding PRs for the other repos when I get a chance

**Review by jamie-o-wilkinson — APPROVED** [review]

**Review by DavidBiddle — APPROVED** [review]

**Pull request merged**